python/mxnet/ndarray/__init__.py

```diff
@@ -17,9 +17,11 @@
 
 """NDArray API of MXNet."""
 
-from . import _internal, sparse, op
-from .op import CachedOp
+from . import _internal, contrib, linalg, random, sparse
 # pylint: disable=wildcard-import, redefined-builtin
+from .op import *
```

Comment:
will doc display properly for operators?

Reply:
Seems the ops cannot be displayed in docs. I will figure out the root cause.